lib/ProductOpener/Config_off.pm

```diff
@@ -565,6 +566,7 @@ improvements
 
 @drilldown_fields = qw(
 	brands
+	brand_owner
```

Comment:
drill down fields can only be facets. I don't think it's useful to make this a facet. (we are close to the 64 facets allowed by mongodb indexes)

Reply:
if we do want to make brand_owner a facet, then we would need to add values in po/tags/

Reply:
fixed